grudge/trace_pair.py

```diff
@@ -195,19 +201,20 @@ def bv_trace_pair(
 
     :arg dd: a :class:`~grudge.dof_desc.DOFDesc`, or a value convertible to one,
         which describes the boundary discretization.
-    :arg interior: a :class:`~meshmode.dof_array.DOFArray` that contains data
+    :arg interior: a :class:`~meshmode.dof_array.DOFArray` or an
+        :class:`~arraycontext.container.ArrayContainer` that contains data
         defined in the volume, which will be restricted to the boundary denoted
         by *dd*. The result will be used as the interior value
         for the flux.
-    :arg exterior: a :class:`~meshmode.dof_array.DOFArray` that contains data
+    :arg exterior: a :class:`~meshmode.dof_array.DOFArray` or an
+        :class:`~arraycontext.container.ArrayContainer` that contains data
         that already lives on the boundary representing the exterior value to
         be used for the flux.
     :returns: a :class:`TracePair` on the boundary.
     """
```

Comment:
Add a note that `interior` and `exterior` must have the same structure if they're containers.

Reply:
Done in https://github.com/inducer/grudge/pull/154/commits/e7478d4abbbcd540936c416004573b2907cfa347